Makefile

```diff
@@ -16,9 +16,11 @@
 
 CGO_ENABLED=0
 GOOS=linux
-TEST_IMAGES=$(shell find -L ./test/test_images ./vendor/knative.dev/serving/test/test_images/grpc-ping ./vendor/knative.dev/serving/test/test_images/multicontainer -mindepth 1 -maxdepth 2 -type f -name "*.go" -exec dirname {} \;)
+TEST_IMAGES=$(shell find -L ./test/test_images ./vendor/knative.dev/serving/test/test_images ./vendor/knative.dev/serving/test/test_images/multicontainer -mindepth 1 -maxdepth 1 -type d)
```

Comment:
IMO, this change will break `test-install` command. That we use to build test binaries for this CI runs. `go install` will probably complain about missing `.go` files in `./multicontainer` dir. I'll try to test it later today.